test/main.sh

```diff
@@ -49,6 +49,7 @@ cleanup() {
     sleep 3
     rm -Rf ${LXD_DIR}
     rm -Rf ${LXD_CONF}
+    rm -f foo.img
```

Comment:
I usually dump everything in ${LXD_DIR} to avoid polluting the current directory and having to clean things up.